tests/conftest.py

```diff
@@ -312,17 +323,20 @@ def _subprocess_setup(request, cfg=None, uwsgi=False):
     from pykern import pkunit
     from pykern import pkio
 
-    # different port than default so can run tests when supervisor running
-    p = "8101"
+    ports = []
```

Comment:
"ports" doesn't need to be saved.

Reply:
[739348c](https://github.com/radiasoft/sirepo/pull/4846/commits/739348cec891a109dbd3ad4390a8c9390d92ba7a)